src/frontend/web_application/src/components/Icon/index.jsx

```diff
@@ -76,7 +78,7 @@ Icon.propTypes = {
 };
 Icon.defaultProps = {
   className: undefined,
-  type: undefined,
+  type: false,
```

Comment:
we defined `type` as a string : when giving default bool it is not coherent in my mind (and I guess react throws a warning).

I think undefined is just fine : we assume it is not set and we can deal with it with a condition same like `false`